docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 web:
-  image: grahamc/jekyll:latest
+  build: .
   ports:
     - "127.0.0.1:4000:4000"
```

Comment:
should not have 127.0.0.1 but rather all ip so that it bind to an ip no matter what your docker setup is.